index.html

```diff
@@ -202,18 +187,23 @@ <h2>
                                 maxlength="30">
                         </li>
                         <li>
-                            <input name="email" type="email" placeholder="Email address" required>
+                            <input name="email" type="email" id="email-address" placeholder="Email address" required>
                         </li>
                         <li>
                             <textarea rows="5" cols="60" name="message" placeholder="Hey I would like to connect to.."
-                                required maxlength="500"></textarea>
+                                required maxlength="500" required></textarea>
                         </li>
                         <li class="button-form">
                             <button type='submit' class="button form-button">Get in touch</button>
+                            <div class="error">
+                                <svg xmlns="http://www.w3.org/2000/svg" fill="darkred" class="bi bi-exclamation-triangle-fill flex-shrink-0 me-2" width="18" height="18" viewBox="0 0 16 16" role="img" aria-label="Warning:">
+                                    <path d="M8.982 1.566a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566zM8 5c.535 0 .954.462.9.995l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995A.905.905 0 0 1 8 5zm.002 6a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"></path>
+                                </svg>
+                                <p>Email must be in lowercase.</p>
+                            </div>
```

Comment:
@muhammaduzairrazaq Hello Uzair nice work there adding a error msg to you contact form! 👍